src/index.js

```diff
@@ -1,4 +1,18 @@
 'use strict';
 
+const _ = require('lodash');
+const Local = require('./cli/domain/local');
+
+function cliCommand(cmd, options, cb) {
+  const dependencies = {
+    local: new Local(),
+    logger: options.logger || { log() {} }
```

Comment:
the logger needs more methods: https://github.com/opentable/oc/blob/master/src/cli/logger.js#L14-L17